docs/contribution_guide/index.rst

```diff
@@ -0,0 +1,233 @@
+.. _contribution_guide:
+
+Contribution Guide
+==================
+
+This project welcomes contributions and suggestions. To find out how to
+connect to our community of users and developers, read our
+:ref:`community pages <community>` including our :ref:`code_of_conduct`.
+The contribution guide explains how to structure your contributions. Please
+:ref:`reach out <communication>` if you encouter any issues.
+
+.. toctree::
+   :maxdepth: 1
+
+   ways_to_contribute
+   contributing_code
+   contributing_example_notebooks
+   fairlearn_proposals
+
+Fairlearn Repository Overview
```

Comment:
I think that "Fairlearn repository overview" and "Development process" should be their own sections. The best place for the two of them is between ways_to_contribute and contributing_code.